com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Motion-Blur.md

```diff
@@ -17,20 +17,25 @@ Motion Blur includes [more options](More-Options.md) that you must manually expo
-| **Camera Motion Blur**             | Indicates whether camera movement contributes to motion blur. Disable this property to stop camera movement from contributing to motion blur. |
+| **Property**              | **Description**                                              |
+| ------------------------- | ------------------------------------------------------------ |
+| **Intensity**             | Set the strength of the Motion Blur effect. This scales the magnitude of the velocities present in the velocity buffer. Set this value to 0 to disable Motion Blur. |
+| **Quality**               | Specifies the quality level to use for this effect. Each quality level applies different preset values. Unity also stops you from editing the properties that the preset overrides. If you want to set your own values for every property, select **Custom**. |
+| **Sample Count**          | Set the maximum number of sample points HDRP uses to compute the Motion Blur effect. Higher values increase the quality and produce a smoother blur. Higher values also increase the resource intensity of the effect. |
+| **Maximum Velocity**      | Use the slider to set the maximum velocity, in pixels, that HDRP allows for all sources of motion blur except Camera rotation. This clamps any value above this threshold to the threshold value. Higher values result in a more intense blur, and an increase in resource intensity. |
+| **Minimum Velocity**      | Use the slider to set the minimum velocity, in pixels, that triggers motion blur. Higher values mean that HDRP does not calculate Motion Blur for slow-moving GameObjects. This decreases the resource intensity. |
+| **Camera Motion Blur**    | Indicates whether camera movement contributes to motion blur. Disable this property to stop camera movement from contributing to motion blur. |
+| **Camera Clamp Mode**     | Determine how the component of the motion vectors coming from the camera is clamped. It is important to remember that clamping the camera component separately, velocities relative to camera might change too (e.g. a GameObject parented to a camera when the camera moves might not have a 0 motion vector anymore). |
+| **- Rotation Clamp**      | Use the slider to set the maximum velocity that HDRP allows Camera rotation to contribute to the velocities of GameObjects. This value is expressed in terms of screen fraction. Higher values result in Camera rotation giving wider blurs. This is only valid if **Camera Clamp Mode** is set to **Rotation** or **Separate Translation And Rotation**. |
+| **- Translation Clamp**   | Use the slider to set the maximum velocity that HDRP allows Camera translation to contribute to the velocities of GameObjects. This value is expressed in terms of screen fraction. Higher values result in Camera rotation giving wider blurs. This is only valid if **Camera Clamp Mode** is set to **Translation** or **Separate Translation And Rotation**. |
```

Comment:
\<br/>This property only appears if you set **Camera Clamp Mode** to **Translation** or **Separate Translation And Rotation**.